index.html

```diff
@@ -294,55 +294,58 @@ <h1 class="title has-text-light" id="documentation">
       </div>
       </div>
       </section> -->
-      <!-- Finance section
-        <section class="hero has-background-light has-text-dark is-medium">
-        <div class="hero-body">
+    <section class="hero has-text-dark is-medium" style="background-color: #B2D3E6">
+      <div class="hero-body">
         <div class="container">
-        <div class="content">
-        <h1 class="title has-text-dark" id="documentation">
-        Finance
-        </h1>
-        <div class="tile is-ancestor">
-        <div class="tile is-parent">
-        <div class="tile is-child">
-        <nav class="level">
-        <div class="level-item has-text-centered">
-        <div>
-        <p class="heading">Current Balance</p>
-        <p class="title has-text-dark">$33,456</p>
-        </div>
-        </div>
-        <div class="level-item has-text-centered">
-        <div>
-        <p class="heading">Income (per month)</p>
-        <p class="title has-text-dark">$132</p>
-        </div>
-        </div>
-        <div class="level-item has-text-centered">
-        <div>
-        <p class="heading">Expenses (per month)</p>
-        <p class="title has-text-dark">$102</p>
-        </div>
-        </div>
-        <div class="level-item has-text-centered">
-        <div>
-        <p class="heading">Current Runway</p>
-        <p class="title has-text-dark">∞ days</p>
-        </div>
-        </div>
-        </nav>
-        </div></div>
-        <div class="tile is-parent">
-        <div class="tile is-child">
-        <p style="width: 400px;">And of course our finance is public too! Then
-        some more text here before publishing this page, so need to fill out
-        why the finance is public here as well.</p>
-        </div></div></div>
-        <p>See more details on our finance website: <a href="#" class="link">https://finance.open-registry.dev</a></p>
-        </div>
-        </div>
+          <div class="content">
+            <h1 class="title has-text-dark" id="documentation">Finance</h1>
```

Comment:
Would have to change the ID attribute here, and probably also uncomment the menu item over at line 48>50